examples/usdt_sample/scripts/latency.c

```diff
@@ -0,0 +1,74 @@
+#include <linux/blkdev.h>
```

Comment:
For this sample, not really. I'm adding more python files which perform similar tasks, and could have reused some of the code in latency.c. I could see a solution, where multiple python files use the same external .c file. No biggie, though, I'll remove the .c file.